mlflow/pyspark/optuna/study.py

```diff
@@ -0,0 +1,188 @@
+import datetime
+import logging
+import sys
+from collections.abc import Callable, Iterable
+from typing import Optional
+
+try:
+    import optuna
+    from optuna import pruners, samplers, storages
+    from optuna.study import Study
+    from optuna.study._optimize import _optimize_sequential
+    from optuna.trial import FrozenTrial
+except ImportError:
+    sys.exit()
+
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import col, first
+
+import mlflow
+from mlflow.exceptions import ExecutionException
+from mlflow.optuna.storage import MlflowStorage
+
+logger = logging.getLogger("optuna-spark")
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```

why not use `__name__`